packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -13,12 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
     > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
 1. Enter values for the **Login redirect URI**. This is the callback from <GuideLink link="../define-callback/">Define a callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/login/callback` or `http://localhost:8080/authentication/login-callback` for Blazor) and production (for example, `https://app.example.com/login/callback`).
-
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```

Comment:
Same as above, for a spa app, Okta cannot reach to the app if app itself is not running in the browser. It has to the the user launch browser and start the flow.

Reply:
@yuliu-okta and @venkatviswanathan-okta 

Should the tooltip text for this field in the Admin Console specify this field is for Web Apps only then? This is the tooltip text that Venkat and you approved:

Include a URI to have Okta initiate the sign-in flow by redirecting to this endpoint. This triggers the client to send an authorize request.

Should we add that it's for browser-based apps only?

Reply:
SPA == browser-based. You mean web app?